### PR TITLE
PIM-3664: Fix product media stacktrace regression on missing media on filesystem

### PR DESCRIPTION
**Pull request by solivier**
Fix product media stacktrace regression on missing media on filesystem during an export

| Q                    | A
| -------------------- | ---
| Bug fix?             | Yes
| New feature?         | No
| BC breaks?           | No
| CI currently passes? | Yes
| Tests pass?          | Yes
| Scenarios pass?      | Yes also green on EE
| Checkstyle issues?*  | No
| PMD issues?**        | No
| Changelog updated?   | Yes
| Fixed tickets        | PIM-3664
| Doc PR               | -

### CODE REVIEW

File: src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ protected function copyMediaFromArray(array $media)
         } else {
             $this->stepExecution->addWarning(
                 $this->getName(),
-                sprintf('Copy of "%s" failed.', $media['filePath']),
+                sprintf('The media is not found or it is not currently available', $media['filePath']),
```
Comment (jjanvier) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php]: had not been found

File: src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php [hunk 2]
```diff
@@ -67,7 +67,7 @@ protected function copyMediaFromArray(array $media)
         } else {
             $this->stepExecution->addWarning(
                 $this->getName(),
-                sprintf('Copy of "%s" failed.', $media['filePath']),
+                sprintf('The media has not been found or is not currently available', $media['filePath']),
```
Comment (filipsalpe) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php]: where did %s go?

Comment (solivier) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php]: it's displayed in the item element

Comment (filipsalpe) [inline src/Pim/Bundle/BaseConnectorBundle/Writer/File/CsvProductWriter.php]: Ok, so no sprintf

**Comment by filipsalpe** [comment]
GTM!

**Pull request merged**